sqlglot/schema.py

```diff
@@ -165,10 +165,25 @@ def column_names(self, table: exp.Table | str, only_visible: bool = False) -> t.
 
     def get_column_type(self, table: exp.Table | str, column: exp.Column) -> exp.DataType.Type:
```

Comment:
`column` can be either an `exp.Column` or a `str`, so we should add the latter here.

Reply:
Indeed, added the str